packages/react/src/themes/teams/components/Popup/popupContentStyles.ts

```diff
@@ -8,6 +8,10 @@ const popupContentStyles: ComponentSlotStylesInput<PopupContentProps, PopupConte
     borderRadius: v.borderRadius,
     display: 'block',
 
+    ...(p.unstable_wrapped && {
+      backgroundColor: 'inherit',
```

Comment:
Wouldn't it be enough to add the styles all the time? without the condition of `p.unstable_wrapped`? I mean maybe we can avoid this prop altogether... Or will adding these styles cause different issue?

Reply:
If we will always add these styles we will inherit `backgroundColor` from `body` when there are no wrapper...